### PR TITLE
[DW-185] Added ResolveLinks script

### PR DESCRIPTION
**Pull request by silrodriguez**

### CODE REVIEW

File: repository-data/application/src/main/resources/hcm-config/configuration/update/ResolveLinks.groovy [hunk 1]
```diff
@@ -0,0 +1,98 @@
+/*
```
Comment (wojtek-oledzki) [inline repository-data/application/src/main/resources/hcm-config/configuration/update/ResolveLinks.groovy]: where is this groovy script used?

File: cms/src/main/java/uk/nhs/digital/ps/migration/ResolveLinksHelper.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
+package uk.nhs.digital.ps.migration;
```
Comment (wojtek-oledzki) [inline cms/src/main/java/uk/nhs/digital/ps/migration/ResolveLinksHelper.java]: this package will be deleted shortly after GOSS migration. If this helper should exist post 19th April, please move it to another package.

**Comment by jacsamell** [comment]
Is this PR still needed?

**Comment by jacsamell** [comment]
This has been lying around for ages, if someone needs this a new PR can be opened from the branch.

**Pull request closed without merging**